docs/pages/workflow/continuous-native-generation.mdx

```diff
@@ -1,115 +1,273 @@
 ---
 title: Continuous Native Generation (CNG)
-description: Learn about Continuous Native Generation (CNG) and how it works.
+description: Learn about managing your native projects with Continuous Native Generation (CNG) and Prebuild.
 sidebar_title: Continuous Native Generation
 ---
 
-import { ContentSpotlight } from '~/ui/components/ContentSpotlight';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
 
-A single native project on its own is complicated to maintain, scale, and update. In a cross-platform app, you have multiple native projects that you must maintain and keep up to date with the latest operating system releases to avoid falling too far behind in any third-party dependencies.
+A single native project on its own is complicated to maintain, scale, and update. In a cross-platform app, you have multiple native projects that you must maintain and keep up to date with the latest operating system releases to avoid falling too far behind in any third-party dependencies. As native your project grows, the complexity from third-party dependencies increases, complicating upgrades and slowing down developer momentum. This discourages adding advanced native functionality and leads to less powerful apps. In cross-platform apps, this complexity is multiplied across each platform.
 
 As your project grows, the complexity involved with each third-party dependency increases and makes the app nearly impossible to upgrade &mdash; this drastically slows down developer momentum, discourages adding complex native functionality, and leads to less powerful apps. To combat these issues, we formalized a process called **Continuous Native Generation (CNG)**.
```

Comment:
this is mostly redundant with the "as your native project grows" part above now.

Reply:
This intention was to keep the second paragraph as is. Sorry for the confusion.